plugins.go

```diff
@@ -41,6 +41,11 @@ func (c *Client) LabelPlugin() (*LabelPlugin, error) {
 	return nil, nil
 }
 
+// GetLabels returns the current labels.
```

Comment:
Shall we be more specific here? e.g. are these all the labels defined across all torrents?

Reply:
This is a list independent of any torrent. Each torrent can only have a label that is one of this list. Perhaps it could be "GetLabels returns a list of the available labels that can be assigned to torrents."

Reply:
Sounds good to me; when in doubt you could always copy from the official RPC API documentation.

Reply:
Yeah I did look for that originally. There's no docs for that particular RPC method https://git.deluge-torrent.org/deluge/tree/deluge/plugins/Label/deluge_label/core.py#n173